inst/extdata/emissions/FUT_EMISS_GV.csv

```diff
@@ -0,0 +1,171 @@
+# File: FUT_EMISS_GV.csv
+# Title: Future atmospheric abundances and climate forcings from scenarios of global and regional hydrofluorocarbon
+# Units: "NA"
```

Comment:
@cahartin um yes so I copied the title of the article as the title of the table. Would a better title be the future global and regional GDP,  emissions, and emission factors for hydrofluorocarbon from ssp scenarios?